internal/tool/kubectl_test.go

```diff
@@ -50,6 +50,9 @@ var output125 = version.Info{
 	Major: "1",
 	Minor: "25",
 }
+
+var latestVersion = output125
```

Comment:
I'd rather do without this declaration here because it is a test file. As the versions change, it would seem we would need to change this variable's value. Is there any other way we can reliably get this value to accomplish this test?

Reply:
I'll have to think more on the latter option. For now, unit testing catching a failure to move this value at least allows us to make sure we don't fail to update the test as the version changes. That should suffice for now.